omnigan/trainer.py

```diff
@@ -441,8 +462,24 @@ def log_comet_images(self, mode, domain):
 
                     if update_task not in save_images:
                         save_images[update_task] = []
+                    if update_task == "m2":
+                        prediction = self.G.decoders["m"](
+                            torch.cat(
+                                (self.z, self.label_2[0, :, :, :].unsqueeze(0)),
```

Comment:
self.label_0[0, :, :, :].unsqueeze(0) should be the same as self.label_0[:1, ...]